catalogue/webapp/utils/search.ts

```diff
@@ -40,6 +40,21 @@ export const getQueryResults = (
     : undefined;
 };
 
+//
+/**
+ * Query properties can an array or a single value.
+ * This returns the first value of the array || string value
+ * @param originalValue - Value from URL query
+ */
+export const getQueryPropertyValue = (
+  originalValue?: string[] | string
+): string =>
+  originalValue
+    ? Array.isArray(originalValue)
+      ? originalValue[0] || ''
+      : originalValue
+    : '';
```

Comment:
We're using the empty string as a default if there's no value specified; my general preference is to lean on the type system for this and use `undefined` as a default, and let the downstream code decide what to do if the user didn't supply a value.